### DISCUSSION TITLE
what is appProjec's orphanedResources used for？I don't know how to use

### DISCUSSION THREAD

**Discussion by 13567436138** [argoproj/argo-cd#12984] [Q&A]
what is appProjec's orphanedResources used for？I don't know how to use

**Comment by reginaelyse** [comment]
hi @13567436138, have you seen the docs on this yet? https://argo-cd.readthedocs.io/en/stable/user-guide/orphaned-resources/
Is there anything specific here you are confused on or want clarified?
